modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -263,6 +263,47 @@ will return a JSON object of the form:
 }
 ```
 
+#### 2.1.6 Single project dicoms tar files
+
+GET /projects/$ProjectName/dicoms
+
+will return a JSON object of the form:
+
+```js
+{
+  "Dicoms": [
+    {
+      "Candidate": "587630",
+      "PSCID": "DCC090",
+      "Entity_type": "Human",
+      "Visit": "V1",
+      "Visit_date": "2018-04-20",
+      "CenterID": "1",
+      "Site": "Data Coordinating Center",
+      "date_acquired": "2018-04-20",
+      "date_first_archived": "2019-05-23 13:46:39",
+      "date_last_archived": "2019-05-23 13:46:39",
+      "tarchiveid": "27",
+      "DicomArchiveID": "1.2.840.113745.101000.1022000.39911.6153.5242769",
+      "Archive": "2018/DCM_2018-04-20_ImagingUpload-13-42-zcoZR0.tar",
+      "Source": "/tmp/ImagingUpload-13-42-zcoZR0",
+      "FileName": "DCM_2018-04-20_ImagingUpload-13-42-zcoZR0.tar"
```

Comment:
is there any reason why the following fields are not returned?
```
AcquisitionCount
DicomFileCount
ScannerManufacturer
ScannerModel
ScannerSoftware
ScannerSerialNumber
```

Those are fields I could see as being informative to the user accessing the DICOM archives.

Reply:
Also, I notice that in the JSON response, keys have all kinds of different convention. I would tend to make them consistent (either stick to CamelCase, or _ separated values, consistent casing etc...). 

Sorry, my brain gets easily distracted with inconsistencies ;).